tests/test_addon_detail.py

```diff
@@ -71,7 +71,7 @@ def test_non_promoted_addon(selenium, base_url, variables):
     # checks the presence of an install warning
     assert addon.install_warning.is_displayed()
     assert variables['install_warning_message'] in \
-        addon.install_warning_message
+           addon.install_warning_message
```

Comment:
This line looks spaced a bit far, also it looks  like a tab with a couple spaces See: https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces